spec/rails_helper.rb

```diff
@@ -21,7 +22,8 @@
         Capybara::Poltergeist::Driver.new(app, js_errors: true,
                                                inspector: true,
                                                phantomjs_options: ['--ignore-ssl-errors=yes'],
-                                               timeout: 120)
+                                               timeout: 120
+        )
```

Comment:
__[rubocop]__
* Align `)` with `(`.
* Closing method call brace must be on the same line as the last argument when opening brace is on the same line as the first argument.

<!-- Comment by [SideCI](https://sideci.com) -->